README.md

```diff
@@ -21,19 +21,19 @@ Masaryk University, Brno, Czech Republic][fimu].
 
 Here are some examples of how you can use the PV211 Utils library:
 
-- First Term Project: Cranfield Collection (20.75% MAP score)
+- First Term Project: Cranfield Collection (23.24% MAP score)
   [![Open in Colab][colab-badge]][cranfield]
   [![Open in Jupyter Hub][jupyter-badge]][jupyter]
 
-- Second Term Project: Beir CQADupStack Collection (17.60% MAP score)
+- Second Term Project: Beir CQADupStack Collection (21.96% MAP score)
   [![Open in Colab][colab-badge]][beir]
   [![Open in Jupyter Hub][jupyter-badge]][jupyter]
 
-- Alternative Second Term Project: ARQMath Collection (0.71% MAP score)
+- Alternative Second Term Project: ARQMath Collection (6.62% MAP score)
```

Comment:
Since the new MAP score (6.62%) passes the lower threshold (1.2%), we need to set a new threshold (see also [how we set the 1.2% threshold in spring 2022][1]).

 [1]: https://gitlab.fi.muni.cz/sojka/mir-general/-/issues/80#note_212773